src/main/java/gregtech/api/metatileentity/WorkableTieredMetaTileEntity.java

```diff
@@ -33,10 +33,13 @@
 
     private final Function<Integer, Integer> tankScalingFunction;
 
+    public final boolean handlesRecipeOutputs;
+
     public WorkableTieredMetaTileEntity(ResourceLocation metaTileEntityId, RecipeMap<?> recipeMap, ICubeRenderer renderer, int tier,
-                                        Function<Integer, Integer> tankScalingFunction) {
+                                        Function<Integer, Integer> tankScalingFunction, boolean handlesRecipeOutputs) {
```

Comment:
You can an extra constructor without the boolean which passed `true` to this one, that way you can undo the changes to the subclasses.